packages/@vue/cli-ui/apollo-server/util/notification.js

```diff
@@ -6,10 +6,15 @@ const builtinIcons = {
   error: path.resolve(__dirname, '../../src/assets/error.png')
 }
 
+let notifCallback = null
```

Comment:
`notifCallback` can be `null`, which is safe because [`node-notifier` defaults it to a no-op](https://github.com/mikaelbr/node-notifier/blob/29b99f2855f09ed3ee573e0410eb4ea4553e10f0/notifiers/toaster.js#L54).